Jenkinsfile

```diff
@@ -748,7 +770,7 @@ def transform(String original, String transform) {
     """
   } else {
     bat """
-      powershell \"\$xml = Resolve-Path ${original};\$output = Join-Path (\$pwd) temp.xml;\$xslt = New-Object System.Xml.Xsl.XslCompiledTransform;\$xslt.Load(\"${transform}\");\$xslt.Transform(\$xml, \$output);\"
+      powershell \"\$xml = Resolve-Path ${original};\$output = Join-Path (\$pwd) temp.xml;\$xslt = New-Object System.Xml.Xsl.XslCompiledTransform;\$xslt.Load(\\\"${transform}\\\");\$xslt.Transform(\$xml, \$output);\"
```

Comment:
Ah... I [guess it's not](https://issues.jenkins-ci.org/browse/JENKINS-33450), which is why we're not using it.